src/data/phases.yaml

```diff
@@ -9,42 +9,29 @@
 # - to add a new item, you'll need to generate a new UUID
 # - you should never replace an item completely while keeping the UUID the same
 # - if you don't know what you're doing, ask!
--
-  id: ac13b0b8-0dc3-44ae-8bb6-0894fd06f0ca
-  number: 1
-  channelName: phase-1
-  hasVoting: false
-  hasRetrospective: false
-  practiceGoalNumber: null
-  interviewGoalNumber: 416
 -
   id: e9548492-6106-4b06-bb54-a42d9479efb1
   number: 2
-  channelName: phase-2
+  name: Foundations
+  channelName: phase-foundations
   hasVoting: false
   hasRetrospective: false
   practiceGoalNumber: null
-  interviewGoalNumber: 417
+  interviewGoalNumber: null
```

Comment:
should `practiceGoalNumber` and `interviewGoalNumber` be false? Your overview indicated they'd be false, but here they're set to null.

Reply:
Typo, perhaps; they're numbers rather than booleans and should be `null`.